components/list/list-item-drag-drop-mixin.js

```diff
@@ -47,8 +47,10 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 			draggable: { type: Boolean, reflect: true },
 			dragging: { type: Boolean, reflect: true },
 			dragHandleText: { type: String, attribute: 'drag-handle-text' },
+			dropDestination: { type: Number, reflect: true },
```

Comment:
multi-word properties should always be translated to kebab-case.
```suggestion
			dropDestination: { type: Number, reflect: true, attribute: 'drop-destination' },
```